server2/README.md

```diff
@@ -0,0 +1,42 @@
+# generate self signed cert and run locally
```

Comment:
This is confusing, what does running all these comments get me (in addition to a self signed cert)

Reply:
I added more documentation.